src/apps/myaccount/app.js

```diff
@@ -471,9 +471,11 @@ define(function(require) {
 								self.displayUserSection();
 
 								myaccount.addClass('myaccount-open');
-								setTimeout(function() { $('#monster_content').hide(); }, 300);
 
-								callback && callback();
+								myaccount.one('transitionend', function() {
```

Comment:
I think there is a small typo here:
```suggestion
								myaccount.on('transitionend', function() {
```